python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -273,6 +277,9 @@ def Index__new__(cls, *args, **kwargs):
         "__new__": Index__new__,
         "_constructor": _FastSlowAttribute("_constructor"),
         "__array_ufunc__": _FastSlowAttribute("__array_ufunc__"),
+        "_accessors": set(),
+        "_data": _FastSlowAttribute("_data", private=True),
```

Comment:
So these attributes we are wanting the fast path to have a chance to evaluate and not automatically be `_Unusable()`?

Reply:
yes

Reply:
Ah so in this case since it's `private` we won't try to operate on e.g. `cudf.Index._data` which is a `ColumnAccessor` and instead reach for `cudf.Index._data` which is a numpy ndarray which we can wrap.

This part of the eager population I'm not too thrilled about. These private variables are not guaranteed to be stable. It would be nice if any slow attribute was attempted to return a proxy object so we didn't have to specify attributes like this but I suppose that can be looked at in a follow up

Reply:
Yeah, there are still such cases we can address in follow-ups. At this point the scope of this PR has kept on expanding, that's why I haven't addressed additional issues in this PR.